BEIMA.Backend/MongoService/IMongoConnector.cs

```diff
@@ -146,18 +146,18 @@ public interface IMongoConnector
         /// <returns>List of all user BsonDocuments</returns>
         public List<BsonDocument> GetAllUsers();
 
-        /// <summary>
```

Comment:
This shouldn't be removed as the login endpoint uses it.

Reply:
I think this is a duplicate method in here which is why the build was failing.